styles/variables-base.less

```diff
@@ -110,7 +110,8 @@
 
 // Checkbox
 // ---------------------------------------
-@agate-checkbox-size: 46px;
+@agate-checkbox-size: inherit;
```

Comment:
Often, for sizes, it's a good idea to leave some value in the base file because the variable may be used for calculations. The value "inherit" isn't a number so the compiler will panic when it attempts to do math on the string. Probably a good idea to leave this at `46px`, or at least some other number.